### ISSUE TITLE
Cannot access files on /sdcard/

### ISSUE THREAD

**Issue by haroon-ali** [green-green-avk/AnotherTermLite#1]
Hi,
At 1st thanks for the only android app that can do something useful on USB/serial.
I'm using android oreo, the app doesn't ask for any storage permission so I cannot access any files under /sdcard.

**Comment by green-green-avk** [comment]
I did not add READ_EXTERNAL_STORAGE and WRITE_EXTERNAL_STORAGE permissions intentionally in order to prevent a reasonable question: "Why a terminal application needs to access my storage?". Technically, they are dangerous. That's why I added $TERMSH pick/send/view/edit commands.

It should be mentioned that there is also $EXTERNAL_DATA_DIR that points to the application dedicated shared storage that is currently available for reading and writing, see https://developer.android.com/reference/android/content/Context.html#getExternalFilesDir(java.lang.String) for more details.

Current picture: I was forced to add these permissions for Android 4.X due to:
a) Requirement of these permissions to access application dedicated shared storage before API 18.
b) Some old devices with API>=18 due to pretty old and bad built-in file managers that provide direct file URIs instead of content provider ones.

Maybe, It's reasonable to enable these permissions for Android >= 6 as it provides a way to turn them off by the user. But be aware of the Google trend to tighten security policies more and more: https://developer.android.com/reference/android/os/Environment.html#getExternalStoragePublicDirectory(java.lang.String)

I'd like to see more opinions before moving forward.
